src/App.tsx

```diff
@@ -3,33 +3,48 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+type State = {
+  query: string,
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    query: '',
+  };
 
   render() {
+    const visibleMovies = moviesFromServer.filter(movie => {
+      const title = movie.title.toLowerCase();
+      const description = movie.description.toLowerCase();
+      const query = this.state.query.toLowerCase();
+
+      return title.includes(query) || description.includes(query);
+    });
```

Comment:
вынеси логику фильтра в отдельный метод
```suggestion
    const visibleMovies = this.getVisibleMovies();
```

Reply:
Done!